models/balancer/ethereum/balancer_v1_ethereum_pools_tokens_weights.sql

```diff
@@ -0,0 +1,101 @@
+{{
+    config(
+        schema = 'balancer_v1_ethereum',
+        alias='pools_tokens_weights',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "balancer",
+                                    \'["metacrypto", "jacektrocinski"]\') }}'
+    ) 
+}}
+
+WITH events AS (
+    -- Binds
```

Comment:
We can add some dates here for default incremental filters
```suggestion
}}
{% set bind_start_date = '2020-02-28' %}
{% set rebind_start_date = '2020-04-01' %}
{% set unbind_start_date = '2020-04-05' %}
WITH events AS (
    -- Binds
```

Reply:
Derived from [queries here](https://dune.com/queries/1747075).